scripts/generate-analysis-files-for-release.sh

```diff
@@ -0,0 +1,123 @@
+# local testing running/testing of all other steps
+RUN_LOCAL=${RUN_LOCAL:-0}
+
+# Get base directory of project
+cd ..
+BASEDIR="$(pwd)"
+cd -
+
+analyses_dir="$BASEDIR/analyses"
+data_dir="$BASEDIR/data"
+scratch_dir="$BASEDIR/scratch"
+
+# Collapsed RNA-seq files
+echo "Create collapse RSEM files"
+bash ${analyses_dir}/collapse-rnaseq/run-collapse-rnaseq.sh
+
+# Create the independent sample list using the *BASE* histology file
+echo "Create independent sample list"
+OPENPBTA_BASE_RELEASE=1 bash ${analyses_dir}/independent-samples/run-independent-samples.sh
+
+# Fusion filtering
+echo "Create fusion filtered list"
+OPENPBTA_BASE_RELEASE=1 bash ${analyses_dir}/fusion_filtering/run_fusion_merged.sh
+
+# Fusion summary
+echo "Run fusion summary for subtypes"
+bash ${analyses_dir}/fusion-summary/run-new-analysis.sh
+
+# Compile all the files that need to be included in the release in one place
+# in the scratch directory
+compiled_dir=${scratch_dir}/analysis_files_for_release
+mkdir -p ${compiled_dir}
+
+# Copy over collapsed RNA-seq files
+cp ${analyses_dir}/collapse-rnaseq/results/pbta-gene-expression-rsem-fpkm-collapsed.stranded.rds ${compiled_dir}
+cp ${analyses_dir}/collapse-rnaseq/results/pbta-gene-expression-rsem-fpkm-collapsed.polya.rds ${compiled_dir}
+
+# Copy over independent specimen lists
+cp ${analyses_dir}/independent-samples/results/independent-specimens.*  ${compiled_dir}
+
+# Copy over fusions lists
+cp ${analyses_dir}/fusion_filtering/results/pbta-fusion-putative-oncogenic.tsv ${compiled_dir}
+cp ${analyses_dir}/fusion_filtering/results/pbta-fusion-recurrently-fused-genes-* ${compiled_dir}
+
+# Copy over fusion summary
+cp ${analyses_dir}/fusion-summary/results/* ${compiled_dir}
+
+# Run modules that cannot be run locally due to memory requirements
+if [ "$RUN_LOCAL" -lt "1" ]; then
+
+  # Run SNV consensus & TMB step for PBTA data
+  echo "Run SNV callers module for PBTA data"
+  OPENPBTA_BASE_RELEASE=1 bash ${analyses_dir}/snv-callers/run_caller_consensus_analysis-pbta.sh
+
+  # Run SNV consensus & TMB step for TCGA data
+  echo "Run SNV callers module for TCGA data"
+  bash ${analyses_dir}/snv-callers/run_caller_consensus_analysis-tcga.sh
+
+  # Copy over SNV callers
+  ## PBTA
+  cp ${analyses_dir}/snv-callers/results/consensus/pbta-snv-consensus-mutation.maf.tsv.gz ${compiled_dir}
+  cp ${analyses_dir}/snv-callers/results/consensus/pbta-snv-mutation-tmb-coding.tsv ${compiled_dir}
+  cp ${analyses_dir}/snv-callers/results/consensus/pbta-snv-mutation-tmb-all.tsv ${compiled_dir}
+  ## TCGA
+  cp ${analyses_dir}/snv-callers/results/consensus/tcga-snv-consensus-snv.maf.tsv.gz ${compiled_dir}
+  cp ${analyses_dir}/snv-callers/results/consensus/tcga-snv-mutation-tmb-coding.tsv ${compiled_dir}
+  cp ${analyses_dir}/snv-callers/results/consensus/tcga-snv-mutation-tmb-all.tsv ${compiled_dir}
+
+  # Run hotspot detection
+  echo "Run hotspots detection"
+  bash ${analyses_dir}/hotspots-detection/run_overlaps_hotspots.sh
```

Comment:
Noting that this module (and output file `pbta-snv-scavenged-hotspots.maf.tsv.gz`) appears to be missing from [here](https://github.com/AlexsLemonade/OpenPBTA-analysis/blob/master/doc/data-files-description.md) but it should be there, right?

Reply:
Yes they should, but maybe that should get fixed on #1424 and not here?

Reply:
I'll open an issue